.github/workflows/swift-toolchain.yml

```diff
@@ -2132,42 +2131,84 @@ jobs:
           - arch: amd64
             cpu: x86_64
             triple: x86_64-unknown-windows-msvc
-
-          # TODO(thebrowsercompany/swift-build#125): Fix the layout of SDK files so we can cross-compile to arm64 
-          # - arch: arm64
-          #   cpu: aarch64
-          #   triple: aarch64-unknown-windows-msvc
+          - arch: arm64
+            cpu: aarch64
+            triple: aarch64-unknown-windows-msvc
 
     steps:
-      - name: Download compilers
+      - name: Download host compilers
         uses: actions/download-artifact@v4
         with:
           name: compilers-amd64
-          path: ${{ github.workspace }}/BuildRoot/Library
+          path: ${{ github.workspace }}/BuildRoot/
 
-      - name: Download devtools
+      - name: Download host devtools
         uses: actions/download-artifact@v4
         with:
           name: devtools-amd64
-          path: ${{ github.workspace }}/BuildRoot/Library
+          path: ${{ github.workspace }}/BuildRoot/
 
-      - name: Download cmark-gfm-amd64
+      - name: Download host Windows runtime
         uses: actions/download-artifact@v4
         with:
-          name: cmark-gfm-amd64-0.29.0.gfm.13
-          path: ${{ github.workspace }}/BuildRoot/Library/cmark-gfm-0.29.0.gfm.13/usr
+          name: windows-sdk-amd64
+          path: ${{ github.workspace }}/BinaryCache/
 
-      - name: Download amd64 Windows Runtime
+      - name: Download host Windows SDK
```

Comment:
```suggestion
      - name: Download Windows SDK (ARM64)
```

Reply:
Why are we downloading it here and then re-downloading it below?

Reply:
(Responded above)